worlds/metroidprime/Items.py

```diff
@@ -48,12 +48,15 @@ class MetroidPrimeItem(Item):
     "Varia Suit": ItemData("Varia Suit", 22, ItemClassification.progression),
     "Phazon Suit": ItemData("Phazon Suit", 23, ItemClassification.progression),
     "Energy Tank": ItemData("Energy Tank", 24, ItemClassification.useful, 14),
-    "UnknownItem1": ItemData("UnknownItem1", 25, ItemClassification.useful),
     # item 026 is a health refill
-    "Ice Trap": ItemData("Ice Trap", 27, ItemClassification.trap),
     "Wavebuster": ItemData("Wavebuster", 28, ItemClassification.filler),
 }
 
+misc_item_table: dict[str, ItemData] = {
```

Comment:
Moved these out so they aren't included in the item pool for now (we'll obviously want to move ice traps back in eventually)